### PR TITLE
react router dom package updated

### LINKED ISSUE #168: upgrage to react-router-dom v6
#### React Router Dom package upgrade.
this application is using version 5 of react-router-dom I would upgrade it to version 6.

### Expected behaviour

React router dom v6

### Screenshots

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by ghost**
# Description
updated react-router-dom from v5 to v6
"react-router-dom": "^6.9.0" 
Fixes:  #168 

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.

---

## ATTACH SCREEN-SHOTS / DEPLOYMENT LINK:
rrd

## Additional miscellaneous futuristic code addition (layout) component
we can add header and footer component inside a Layout component and remove them from App.js file

### CODE REVIEW

File: src/components/layout/Layout.js
Comment (aklite) [inline src/components/layout/Layout.js:1]: we could make the App.js more clean to see 

one of the best practice i saw good developers do is this by making a layout .js file. for setting headers and footers

**Review by Jai-Solania-29 — APPROVED** [review]
LGTM 🚀

**Comment by Jai-Solania-29** [comment]
Hey, @aklite Thanks for contributing, Great Work 🚀
Merging your PR 🎉
Don't forget to give a ⭐ to the repo 😃

**Comment by ghost** [comment]
Thanks @Jai-Solania-29 !! 😁

I am looking forward to contribute to this project it  could have look at my next issue related to layouts 😊

Already starred 😁

**Pull request merged**